HW_7/task7.2.py

```diff
@@ -0,0 +1,6 @@
+def print_operation_table(operation, num_rows=6, num_columns=6):
+    for i in range(1, num_rows+1):
+        print(*list(map(lambda j: operation(i, j), range(1, num_columns+1))), sep="\t")
```

Comment:
Очень изобретательно. Браво!
```suggestion
        print(*map(lambda j: operation(i, j), range(1, num_columns+1)), sep="\t")
```